js_modules/dagit/packages/core/src/graph/SVGViewport.tsx

```diff
@@ -115,16 +117,32 @@ const PanAndZoomInteractor: SVGViewportInteractor = {
 
   onWheel(viewport: SVGViewport, event: WheelEvent) {
     const cursorPosition = viewport.getOffsetXY(event);
+    const flagDAGPanWithScrollWheel = featureEnabled(FeatureFlag.flagDAGPanWithScrollWheel);
+
+    // convert wheel event units to a better scroll speed
+    const speed = 0.7;
+
     if (!cursorPosition) {
       return;
     }
-
-    if (event.altKey || event.shiftKey) {
-      viewport.shiftXY(-event.deltaX, -event.deltaY);
+    if (flagDAGPanWithScrollWheel) {
```

Comment:
Shouldn't this flag only be around the new behavior? Looks like if the flag is off then `onWheel` is a no-op. is that correct?

Reply:
Ahh the last  `} else {` is for this flag so we should be good on that front!

Reply:
oh derp, I can't count closing/openning braces 😅